android/src/main/kotlin/com/signify/hue/flutterreactiveble/PluginController.kt

```diff
@@ -77,7 +78,7 @@ class PluginController {
         https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling
         */
         RxJavaPlugins.setErrorHandler { throwable ->
-            if (throwable is UndeliverableException && throwable.cause is BleException) {
+            if (throwable is UndeliverableException && (throwable.cause is BleException) || throwable.cause is RuntimeException) {
```

Comment:
I don't think parentheses around `throwable.cause is BleException` are required.

What's the rationale behind adding `throwable.cause is RuntimeException`?

Do you mean what is written? The grouping is as follows: `(throwable is UndeliverableException && throwable.cause is BleException) || throwable.cause is RuntimeException)`.

Reply:
whoops indeed this will provide nasty side effects s oremoved it